packages/integration-sdk-runtime/src/synchronization/index.ts

```diff
@@ -426,6 +437,94 @@ export async function uploadData<T extends UploadDataLookup, K extends keyof T>(
   );
 }
 
+// Interface for storing both the key value and total size of a given array entry
+interface keyAndSize {
```

Comment:
I haven't seen camel case used for interface naming. Have you seen this? I've only seen pascal case used for `interface` names and camel case used for `interface` members.

Reply:
Thanks for the catch!  Updated in latest commit.